examples/camera-app/linux/src/clusters/webrtc_provider/webrtc-provider-manager.cpp

```diff
@@ -31,6 +31,50 @@ using namespace chip::app::Clusters::WebRTCTransportProvider;
 
 using namespace Camera;
 
+void WebRTCProviderManager::Init()
+{
+    rtc::Configuration config;
+    // config.iceServers.emplace_back("stun.l.google.com:19302");
+
+    mPeerConnection = std::make_shared<rtc::PeerConnection>(config);
+
+    mPeerConnection->onLocalDescription([this](rtc::Description description) {
+        mSdpAnswer = std::string(description);
+        ChipLogProgress(Camera, "Local Description:");
+        ChipLogProgress(Camera, "%s", mSdpAnswer.c_str());
+
+        ScheduleAnswerSend();
```

Comment:
Why is it only sending an Answer when it has a local SDP? What path would it take when it receives a SolicitOffer and has to send an Offer to the controller?

Reply:
SolicitOffer flow has not been implemented yet. 

This is triggered after the camera receives the ProvideOffer command, and call mPeerConnection->setRemoteDescription(args.sdp); to install the remote offer, and then onLocalDescription callback is called, we schedule to send the Answer command here.